### DISCUSSION TITLE
Vercel ai sdk

### DISCUSSION THREAD

**Discussion by patrickwasp** [zauberzeug/nicegui#1043] [Ideas / Feature Requests]
Vercel recently released a library for integrating with chat apis and building interactive UIs. 

https://vercel.com/blog/introducing-the-vercel-ai-sdk
https://github.com/vercel-labs/ai

they have examples for Nuxt which could give some ideas of how to integrate with nicegui.
https://github.com/vercel-labs/ai/blob/main/examples/nuxt-openai/app.vue

**Comment by kodsher** [comment]
Have you figured out how to launch a simple nicegui template app with vercel?

**Reply by modfinpython** [reply]
Anyone?
